sources/packages/backend/workflow/test/2021-2022/parttime-assessment/costs/parttime-assessment-2021-2022-costs-child-care-cost.e2e-spec.ts

```diff
@@ -0,0 +1,211 @@
+      // Creates 2 eligible dependents.
+      assessmentConsolidatedData.studentDataDependants = [
+        createFakeStudentDependentEligibleForChildcareCost(
+          DependentChildCareEligibility.Eligible0To11YearsOld,
+          assessmentConsolidatedData.offeringStudyStartDate,
+        ),
+        createFakeStudentDependentEligibleForChildcareCost(
+          DependentChildCareEligibility.Eligible12YearsAndOver,
+          assessmentConsolidatedData.offeringStudyStartDate,
+        ),
+      ];
+      const totalChildcareCostEnteredInApplication =
+        assessmentConsolidatedData.studentDataDaycareCosts11YearsOrUnder +
+        assessmentConsolidatedData.studentDataDaycareCosts12YearsOrOver;
+      // Act
+      const calculatedAssessment =
+        await executePartTimeAssessmentForProgramYear(
+          PROGRAM_YEAR,
+          assessmentConsolidatedData,
+        );
+      // Assert
+      // Childcare cost for the category 11 years and under must be equal to the value entered in application
+      // for this category as student has one or more dependent in this category.
+      expect(
+        calculatedAssessment.variables.calculatedDataDaycareCosts11YearsOrUnder,
+      ).toBe(assessmentConsolidatedData.studentDataDaycareCosts11YearsOrUnder);
+      // Childcare cost for the category 12 years and over under must be equal to the value entered in application
+      // for this category as student has one or more dependent in this category.
+      expect(
+        calculatedAssessment.variables.calculatedDataDaycareCosts12YearsOrOver,
+      ).toBe(assessmentConsolidatedData.studentDataDaycareCosts12YearsOrOver);
+      // Child care cost expected to be the total sum of values entered in student application.
+      expect(calculatedAssessment.variables.calculatedDataChildCareCost).toBe(
+        totalChildcareCostEnteredInApplication,
+      );
+      // As the total sum of child care costs are beyond the maximum allowable limit
+      // the maximum limit must be considered towards assessed need.
+      expect(
+        calculatedAssessment.variables.calculatedDataTotalChildCareCost,
+      ).toBe(1822.4);
+      expect(
+        calculatedAssessment.variables.calculatedDataTotalChildCareCost,
+      ).toBeLessThan(
+        calculatedAssessment.variables.calculatedDataChildCareCost,
+      );
+    },
+  );
+
+  it(
+    "Should calculate child care costs as 0 when student has one or more dependents on both categories(11 years or under and over 12 years " +
+      " or over and declared on taxes for disability) but child care costs not provided in student application.",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
+        createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+      assessmentConsolidatedData.offeringWeeks = 8;
+      assessmentConsolidatedData.offeringCourseLoad = 20;
+      // Creates 4 eligible and 4 not eligible dependents.
```

Comment:
Shouldn't it be "1 eligible and 1 not eligible" ?

Reply:
It should be `// Creates 2 eligible dependents.` I will change it.